src/ert/com_protocol/_schema.proto

```diff
@@ -54,19 +84,20 @@ message JobId {
 
 message Experiment {
   required ExperimentId id   = 1; // NOTE: in the message the id is optional since it often will be streamed
-  optional Status status   = 2 [default = UNKNOWN];
+  optional ExperimentStatus status   = 2 [default = EXPERIMENT_STARTED];
```

Comment:
No `black` for proto files it seems :)  My OCD keeps my eyes locked at that extra whitespace..

Reply:
`clang-format` can format `.proto` files so will use it.